build.sbt

```diff
@@ -43,13 +48,53 @@ bashScriptExtraDefines += "addJava \"-Dpidfile.path=/var/run/"+(packageName in L
 bashScriptExtraDefines += "addJava \"-Ddb.default.url=jdbc:derby:$DP2DATA/db;create=true\""
 com.typesafe.sbt.packager.SettingsHelper.makeDeploymentSettings(Debian, packageBin in Debian, "deb")
 com.typesafe.sbt.packager.SettingsHelper.makeDeploymentSettings(Universal, packageBin in Universal, "zip")
-
 resolvers += Resolver.url("Typesafe Ivy releases", url("https://repo.typesafe.com/typesafe/ivy-releases"))(Resolver.ivyStylePatterns)
 resolvers += "Sonatype OSS Releases" at "https://oss.sonatype.org/content/repositories/releases/"
 //resolvers += "Sonatype OSS Staging" at "https://oss.sonatype.org/content/repositories/staging/"
 resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
 resolvers += "Local Maven Repository" at Path.userHome.asFile.toURI.toURL+".m2/repository/"
 
+// Documentation for RPM packaging with sbt-native-packager available at:
+// <http://www.scala-sbt.org/sbt-native-packager/formats/rpm.html>
+// For packaging on Linux (CentOS/Redhat "RPM" flavor)
+// Informational, dependency, meta, scriptlet, systemV start and script settings
+packageName in Rpm := "pipeline2-webui"
+version in Rpm := "1.8.1"
+rpmRelease := Option(sys.props("rpm.buildNumber")) getOrElse "1"
+daemonUser in Linux := "tpbadmin"
+daemonGroup in Linux := (daemonUser in Linux).value
+packageArchitecture in Rpm := "noarch"
+packageSummary in Rpm := "DAISY Pipeline 2 Web User Interface."
+packageDescription in Rpm := "A web-based user interface for the DAISY Pipeline 2."
+rpmVendor := "The DAISY Consortium"
+rpmUrl := Option("https://github.com/daisy/pipeline-webui")
+rpmLicense := Option("LGPLv3")
+executableScriptName := "pipeline2-webui"
+rpmAutoreq += "java8-runtime"
+rpmAutoreq += "pipeline2"
+serverLoading in Linux := Upstart 
+maintainerScripts in Rpm := Map(
+  Pre -> Seq("""echo "pre-install""""),
+  Post -> Seq("""rm -rf /opt/daisy-pipeline2-webui/webui/dp2webui && cp -rf /opt/daisy-pipeline2-webui/webui/db-empty /opt/daisy-pipeline2-webui/webui/dp2webui && rm -rf /opt/daisy-pipeline2-webui/webui/db-empty && service daisy-pipeline2-webui start"""),
```

Comment:
@alaram are you sure this is correct? If you look at the `bashScriptExtraDefines`-lines they add some bash script lines to the webui start script that creates the database if it's not present. The database is expected to be stored in `$DP2HOME/data/db` (i.e. `/opt/daisy-pipeline2-webui/data/db`), and `$DP2HOME/data` is symlinked to `/var/opt/daisy-pipeline2-webui` so that the actual db will be stored in `/var/opt/daisy-pipeline2-webui/db`. There is a `webui` subdirectory in the desktop version of Pipeline 2 v1.9; maybe you've mixed it with that?